symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkApiClientsConfig.java

```diff
@@ -92,12 +93,18 @@ public ConnectionApi connectionApi(@Qualifier("podApiClient") ApiClient podApiCl
     return new ConnectionApi(podApiClient);
   }
 
-  @Bean
+  @Bean(name = "datafeedApi")
+  @Primary
   @ConditionalOnMissingBean
   public DatafeedApi datafeedApi(@Qualifier("datafeedAgentApiClient") ApiClient agentApiClient) {
```

Comment:
Should we provide a way to disable the DF loop then ? If you are using Datahose only?